src/widgets/helper/ChannelView.cpp

```diff
@@ -160,9 +298,18 @@ ChannelView::ChannelView(BaseWidget *parent, Split *split, Context context,
 
     this->pauseTimer_.setSingleShot(true);
     QObject::connect(&this->pauseTimer_, &QTimer::timeout, this, [this] {
-        /// remove elements that are finite
+        // remove elements that are finite
         for (auto it = this->pauses_.begin(); it != this->pauses_.end();)
-            it = it->second ? this->pauses_.erase(it) : ++it;
+        {
+            if (it->second)
+            {
+                it = this->pauses_.erase(it);
+            }
+            else
+            {
+                it = ++it;
+            }
+        }
```

Comment:
Seems like `std::remove_if(..., [](const auto &p) { return p.has_value(); })` (or just `p`).

Reply:
Fixed in [`b9b7137` (#4926)](https://github.com/Chatterino/chatterino2/pull/4926/commits/b9b7137fa8971413f4237468546129455ede6d30)